packages/themes/src/MenderNext/common.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright 2025 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+import { blue, cyan, grey, purple } from '@mui/material/colors';
+
+// breakpoints, metadata, shape config, spacing are not adjusted in the theme
+import colorDefinitions from './figma/material_colors.json' with { type: 'json' };
+import typographyDefinitions from './figma/typography.json' with { type: 'json' };
+
+const typographyTokens = typographyDefinitions.collections.find(({ name }) => name === 'typography');
+const colorTokens = colorDefinitions.collections.find(({ name }) => name === 'material/colors');
+
+export const typography = {
+  fontFamily: typographyTokens?.variables.fontFamily.values['Mode 1'],
+  body1: {
+    fontFamily: typographyTokens?.variables['fontFamily (body)'].values['Mode 1']
+  }
+};
+
+export const colors = {
+  grey: {
+    ...grey,
+    50: colorTokens?.variables['grey (M)']['50'].values['Mode 1'] || grey[50],
+    100: colorTokens?.variables['grey (M)']['100'].values['Mode 1'] || grey[100],
+    200: colorTokens?.variables['grey (M)']['200'].values['Mode 1'] || grey[200],
+    300: colorTokens?.variables['grey (M)']['300'].values['Mode 1'] || grey[300],
+    400: colorTokens?.variables['grey (M)']['400'].values['Mode 1'] || grey[400],
+    500: colorTokens?.variables['grey (M)']['500'].values['Mode 1'] || grey[500],
+    600: colorTokens?.variables['grey (M)']['600'].values['Mode 1'] || grey[600],
+    700: colorTokens?.variables['grey (M)']['700'].values['Mode 1'] || grey[700],
+    800: colorTokens?.variables['grey (M)']['800'].values['Mode 1'] || grey[800],
+    900: colorTokens?.variables['grey (M)']['900'].values['Mode 1'] || grey[900]
```

Comment:
Should we also include accent shade colors? (`A100` - `A700`)

Reply:
Good question - these were unchanged from the MUI defaults and I opted to be explicit both in the color assignment and with the repeatable assignments to have the mapping documented as clear as possible & hopefully extendable for anyone if wanted.
The token sync should ideally give an idea about the granularity and frequency of changes to the palette etc. - but so far I'm not sure if we should try to translate everything from the tokens (potentially leading tying us to Figma specific structures and breaking MUI automatic adjustments for e.g. colors) or mapping as little as possible to  have changes more explicit in order to get a reliable coverage of the theme settings in here (and less adjustments in the products). 
What do you think?

Reply:
I don't have strong opinion, so we can leave as is and see if it works for us  :+1: